editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1163,25 +1163,25 @@ bool CanvasItemEditor::_gui_input_zoom_or_pan(const Ref<InputEvent> &p_event, bo
 	if (k.is_valid()) {
 		if (k->is_pressed()) {
 			if (ED_GET_SHORTCUT("canvas_item_editor/zoom_3.125_percent")->matches_event(p_event)) {
-				_update_zoom((1.0 / 32.0) * MAX(1, EDSCALE));
+				_shortcut_zoom_set((1.0 / 32.0) * MAX(1, EDSCALE));
```

Comment:
Maybe the `* MAX(1, EDSCALE)` can be moved to the function? (Like it was in `3.x`.)